public/app/plugins/datasource/prometheus/mode-prometheus.js

```diff
@@ -50,6 +50,9 @@ var PrometheusHighlightRules = function() {
       token : "keyword.control",
       regex : "by|without|on|ignoring|group_left|group_right",
       next  : "start-label-list-matcher"
+    }, {
+      token : "variable.other",
+      regex : "\$[A-Za-z0-9_]+"
```

Comment:
Seems, `"\$[A-Za-z0-9_]+"` regex should be `"\\$[A-Za-z0-9_]+"` (double slash, because it's a string, not JS regex literal).

Reply:
Thanks, fix regex and token type.